### DISCUSSION TITLE
broadcast failed because of: double spend: -25: bad-txns-inputs-missingorspent

### DISCUSSION THREAD

**Discussion by jonthdiaz** [lightningnetwork/lnd#8493] [Troubleshooting]
I forced some channels to close and I tried to bumpfeeclose. I tried to find the transaction on the mempol but they did not appear show I checked the lnd logs and the error is broadcast failed because of: double spend: -25: bad-txns-inputs-missingorspent

**Comment by bitromortac** [comment]
Do you still have that problem? Have your force close transactions confirmed already?
